qemu/tests/cfg/virtio_port_login.cfg

```diff
@@ -7,6 +7,7 @@
     virtio_port_type_vc1 = "virtio_console"
     shell_cmd_list = " \ls -l , cat /proc/cmdline, ifconfig -a, mkdir -p /tmp/test, rm -rf /tmp/test"
     image_snapshot = yes
+    clean_cmd = ""
```

Comment:
Not necessary, you can set default value in virtio_port_login.py line 29, like `clean_cmd = self.params.get("clean_cmd", "")`